scripts/A4.R

```diff
@@ -0,0 +1,35 @@
+# create ds object --------------------------------------------------------
+
+ds <- create_dataset(id = "A4")
+
+
+# download the data -------------------------------------------------------
+
+ds <- download_data(ds)
+
+# read in the spatial data ------------------------------------------------
+
+spatial_unit_df <- readr::read_csv("data/const/spatial_unit_postgres.csv")
+
+# data cleaning -----------------------------------------------------------
+
+### add spatial unit column "Switzerland"
+ds$data %>%
+  janitor::clean_names() %>%
+  tibble::as_tibble() %>%
+  dplyr::rename("beobachtungseinheit" = rubrik,
+                "anzahl" = tj) -> ds$data
```

Comment:
It should be captured somewhere what the unit means "TJ" stands for "Terajoules" so either this has to be in the tablename or in the columnname.

Reply:
agree! comment below ingest into db should remind us or if the ZHAW approves renaming the "anzahl" to "Terajoules" I'll implement this